test/verify/check-login

```diff
@@ -330,6 +330,42 @@ account    required     pam_succeed_if.so user ingroup %s""" % m.get_admin_group
                                     "couldn't parse login input: Malformed input",
                                     "couldn't parse login input: Authentication failed")
 
+    @skipImage("No SELinux", "debian-stable", "debian-testing", "ubuntu-stable", "ubuntu-1804")
+    @skipImage("Only supported for Fedora >= 30", "fedora-atomic")
+    def testSELinuxRestrictedUser(self):
+        m = self.machine
+        b = self.browser
+
+        # non-admin user_u
+        m.execute("useradd unpriv; echo 'unpriv:foobar' | chpasswd; semanage login -a -s user_u unpriv")
+        m.start_cockpit()
+        b.login_and_go("/system", user="unpriv")
+        # not an admin
+        b.wait_not_present("#shutdown-group button:not(.disabled)")
```

Comment:
I am just wondering if this check is enough. Wouldn't this succeed even before the page would properly load and `#shutdown-group button` would not be present yet?
I feel like `wait_attr("#shutdown-group button", "disabled", "disabled")` (or whatever value it has) or `wait_visible("#shutdown-group button:disabled")` would be safer and better readable (not-present and not-disabled == present and disabled)

Reply:
Good point! This is jquery, so it's always present, but indeed we may soon change the page to start with `disabled` and enable it when there's enough privilege (issue #12242). So this should indeed first wait for the buttons to be present and at least one disabled, and *then* ensure that no button is enabled.

Reply:
Fix pushed. PTAL?